src/modules/utils/motordrivercontrol/MotorDriverControl.cpp

```diff
@@ -113,8 +116,9 @@ bool MotorDriverControl::config_module(uint16_t cs)
         tmc26x= new TMC26X(std::bind( &MotorDriverControl::sendSPI, this, _1, _2, _3), axis);
 
     }else{
-        THEKERNEL->streams->printf("MotorDriverControl %c ERROR: Unknown chip type: %s\n", axis, str.c_str());
-        return false;
+		chip=SPIDRVR;
```

Comment:
tab

Reply:
All the tabs are fixed.....just haven't re-committed yet.  When I do, at some point, would it be best if I rebased and collapsed down to one commit?